server/tests/rest/test_work_shift.py

```diff
@@ -0,0 +1,50 @@
+"""Tests for work shift related REST endpoints."""
+
+import json
+from application.app import create_app
+
+
+def test_list_work_shifts():
+    app = create_app('testing')
+    client = app.test_client()
+    headers = {
+        'Authorization': 'volunteer@slu.edu'
+    }
+    response = client.get('/shifts', headers=headers)
+    data = json.loads(response.data)
+
+    assert response.status_code == 200
+    assert len(data) == 2
+
+
+def test_add_work_shifts():
+    app = create_app('testing')
+    client = app.test_client()
+    new_shift = {
+        'code': 'f853578c-fc0f-4e65-81b8-566c5dffa35d',
+        'worker': 'volunteer@slu.edu',
```

Comment:
> worker information should not be included here. Technically, the code should not be included either (see issue #22), but since issue #22 is not fixed, the test case won't pass without the code. So leave the code in for now, I'll make a note in issue #22 to adjust the test when the issue is resolved.

@Sai9797 You didn't address this comment.